### PR TITLE
6 edit personal checkin data

### PR DESCRIPTION
**Pull request by hey-johnnypark**
This is the backend implementation for when a guest changes its data. 

Highlights:

1) We use papertrail to keep a record of a ticket history.

2) For this similar to active storage we needed to change the relation data type to UUID. 

3) Everything is covered by tests. 

closes https://github.com/railslove/recover-backlog/issues/6

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -107,8 +107,8 @@
     t.datetime "block_at"
     t.bigint "frontend_id"
     t.string "api_token"
-    t.integer "auto_checkout_minutes"
     t.string "menu_alias"
+    t.integer "auto_checkout_minutes"
```
Comment (salimhb) [inline db/schema.rb:111]: Stop trolling @m0ppers 🤣 https://github.com/railslove/rcvr-api/pull/68

**Review by salimhb — CHANGES_REQUESTED** [review]

File: db/schema.rb (hunk 1 above)
Comment (hey-johnnypark) [reply db/schema.rb:111]: What should I do here @salimhb sorry I do not get it :)

Comment (salimhb) [reply db/schema.rb:111]: Don't add this change. @m0ppers already fixed it for the **second** time in #68 😉

**Review by salimhb — APPROVED** [review]

**Pull request merged**